spec/lib/efile/md/md502_calculator_spec.rb

```diff
@@ -818,16 +818,39 @@
     end
   end
 
+  describe "#calculate_line_14" do
+    before do
+      allow_any_instance_of(Efile::Md::TwoIncomeSubtractionWorksheet).to receive(:calculate_line_7).and_return 1000
+    end
+
+    context 'MFJ filing status' do
+      let(:filing_status) { 'married_filing_jointly' }
+      it 'returns the amount from the worksheet if MFJ filing status' do
+        instance.calculate
+        expect(instance.lines[:MD502_LINE_14].value).to eq 1000
+      end
+    end
+
+    context 'MFS filing status' do
+      let(:filing_status) { 'married_filing_separately' }
+      it 'returns the 0 if not MFJ filing status' do
+        instance.calculate
+        expect(instance.lines[:MD502_LINE_14].value).to eq 0
+      end
+    end
+  end
+
   describe "#calculate_line_15" do
     before do
       intake.direct_file_data.total_qualifying_dependent_care_expenses = 2
```

Comment:
[dust] since we're now relying on numbers to calculate these, it could make sense to also stub the lines here as opposed to setting the values for `intake.direct_file_data.total_qualifying_dependent_care_expenses` and `intake.direct_file_data.fed_taxable_ssb`

Reply:
the update was to set the line in the calculator using those fields, but not to add the `calculate_line_x` methods, so the way this is set up is still correct